src/main/java/org/folio/rest/domain/FeeFineNoticeContext.java

```diff
@@ -44,7 +46,9 @@ private boolean isCharge() {
 
   private boolean isFeeFineActon() {
     return feefineaction.getTypeAction().equals(PAID_FULLY) ||
-      feefineaction.getTypeAction().equals(PAID_PARTIALLY);
+      feefineaction.getTypeAction().equals(PAID_PARTIALLY) ||
+      feefineaction.getTypeAction().equals(WAIVED_FULLY) ||
+      feefineaction.getTypeAction().equals(WAIVED_PARTIALLY);
```

Comment:
Maybe we should introduce a constant list of all fee/fine actions and replace this method's body with simple `List#contains` invocation? Keep in mind that this list is only going to grow in subsequent tasks.

Reply:
Done